raiden-ts/src/raiden.ts

```diff
@@ -425,6 +448,20 @@ export class Raiden {
     return this.deps.provider.blockNumber || (await this.deps.provider.getBlockNumber());
   }
 
+  public config(newConfig: Partial<RaidenConfig>) {
+    this.deps.config$
+      .pipe(
+        first(),
+        map((currentConfig: RaidenConfig) => ({ ...currentConfig, ...newConfig })),
```

Comment:
No need to map and then again on subscribe. I think in the subscribe callback is the best place for it, so you can take this map out

Reply:
I guess I missed that one after trying something out and then forgot to remove it, will fix